### DISCUSSION TITLE
Using trained model with Skorch

### DISCUSSION THREAD

**Discussion by btvd** [skorch-dev/skorch#1066] [Q&A] [answered]
Community, is there a way to configure skorch wrapper for PyTorch trained model (no need to use fit method) to add only compatibility with sklearn ecosystem?

**Comment by BenjaminBossan** [comment, marked as answer]
Yes, that's absolutely possible. For instance, check [this notebook](https://github.com/skorch-dev/skorch/blob/master/notebooks/Hugging_Face_Finetuning.ipynb). It defines a `BertModule` that wraps a Hugging Face transformers model. The wrapped model can be used with skorch as usual, whether it be for fine-tuning or just for inference.

**Reply by btvd** [reply]
Thank you very much for hint!
